sopel/modules/wiktionary.py

```diff
@@ -13,13 +13,15 @@
 from sopel import web
 from sopel.module import commands, example
 
-uri = 'https://en.wiktionary.org/w/index.php?title=%s&printable=yes'
+uri = 'http://en.wiktionary.org/w/index.php?title=%s&printable=yes'
```

Comment:
This reverts #1373… Can we not do that? :smile_cat:

Reply:
Gotta remember to check my diffs for unintended changes :confused: